tsserver/tsc.js

```diff
@@ -31087,4 +31087,4 @@ var ts;
     }
 })(ts || (ts = {}));
 ts.executeCommandLine(ts.sys.args);
-//# sourceMappingURL=file:///C:/Users/Zhengbo/Documents/GitHub/TypeScript/built/local/tsc.js.map
```

Comment:
Updated #243. It is important fix for issue #244, which is already in master and cherry-picked here.